tiatoolbox/models/architecture/hovernet.py

```diff
@@ -42,6 +42,100 @@
 from tiatoolbox.utils.misc import get_bounding_box
 
 
+def get_instance_info(pred_inst, pred_type=None):
```

Comment:
Refer to HoVerNet __proc_np_hv and do the same for this. This should not be here in the HoVeNet but be in the HoVerNet as this is specific to HoVerNet

Reply:
Changed to static protected method as part of HoVerNet as discussed.